foodsaving/users/templates/activation-mail.html

```diff
@@ -1,3 +1,5 @@
+{% load i18n %}
+{% blocktrans with display_name=user.display_name %}
```

Comment:
Just FYI, here's our project: https://www.transifex.com/yunity-1/foodsaving-tool/dashboard/
It basically helps managing the translation files, we don't need to edit those *.po files manually.